example/multilang/spoofaxcore/minisdf.spoofaxcore/build.gradle.kts

```diff
@@ -20,3 +21,18 @@ ecj {
 tasks.withType<JavaCompile> { // ECJ does not support headerOutputDirectory (-h argument).
   options.headerOutputDirectory.convention(provider { null })
 }
+
+afterEvaluate {
```

Comment:
What is does is making the interface available for the language(s) to compile with. Ideally, I would like to model it with a regular source dependency, instead of this syncing, but that does not seem to be supported for Statix. Probably that is because compiling with a source dependency would result in missing modules at runtime for regular Spoofax 2 languages.

But I'm very much open for suggestions for improvements a this point.

Reply:
I see, this is a Spoofax 2 language using the Spoofax 2 Gradle plugin. Then this is fine for now.